### PR TITLE
Asya Kolodko test task - Modifying several docs

### PR DESCRIPTION
**Pull request by AsyaKolodko**
### Motivation and context
Here is my take on how to improve the instructions.

### How has this been tested?
By me and spellcheckers.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by nmanovic** [comment]
@AsyaKolodko , thanks for the contribution! We are happy to collaborate with you. I hope to speak with you soon.

A couple of problems that I have found:

1. Tabs are great, but the first item is shifted and the whole text is escaped. Could you please look?
image

File: site/content/en/docs/administration/basics/installation.md [hunk 1]
```diff
@@ -47,31 +44,28 @@ For access from China, read [sources for users from China](#sources-for-users-fr
   sudo apt-get update
   sudo apt-get --no-install-recommends install -y docker-ce docker-ce-cli containerd.io
   ```
-
-- Perform [post-installation steps](https://docs.docker.com/install/linux/linux-postinstall/)
-  to run docker without root permissions.
+More instructions can be found [here](https://docs.docker.com/install/linux/docker-ce/ubuntu/).
+3. Perform [post-installation steps](https://docs.docker.com/install/linux/linux-postinstall/) to run docker without root permissions.
```
Comment (nmanovic) [inline site/content/en/docs/administration/basics/installation.md:48]: Need to have an empty line.

image

**Comment by nmanovic** [comment]
@AsyaKolodko , in general, I like the simplification. But it looks like you didn't check your changes. Please run `hugo server` to see what you will have.

Another problem is the markdown linter. Could you please look at the report: https://github.com/opencv/cvat/suites/8699490759/artifacts/392673460?

**Comment by AsyaKolodko** [comment]
@nmanovic Thank you for the feedback! I fixed the tabs and the rest, as well as I checked it with `hugo server`.

**Pull request closed without merging**